core/modules/system/system.install

```diff
@@ -67,7 +67,7 @@ function system_requirements($phase) {
     $requirements['php'] = array(
       'title' => $t('PHP'),
       'value' => $phpversion,
-      'description' => $t('The phpinfo() function has been disabled for security reasons. To see your server\'s phpinfo() information, change your PHP settings or contact your server administrator. For more information, <a href="@phpinfo">Enabling and disabling phpinfo()</a> handbook page.', array('@phpinfo' => 'http://drupal.org/node/243993')),
+      'description' => $t('The phpinfo() function has been disabled for security reasons. To see your server\'s phpinfo() information, change your PHP settings. For more information, <a href="@phpinfo">Enabling and disabling phpinfo()</a> handbook page.', array('@phpinfo' => 'https://backdropcms.org/user-guide/enabling-and-disabling-phpinfo')),
       'severity' => REQUIREMENT_INFO,
```

Comment:
One possibility to get rid of all escaping and remove all markup from the translatable string:

`$doclink = l(t('Enabling and disabling phpinfo()'), 'https://backdropcms.org/user-guide/enabling-and-disabling-phpinfo');`

and then:

```
      'description' => $t("The phpinfo() function has been disabled for security reasons. To see your server's phpinfo() information, change your PHP settings. For more information see the !doclink handbook page.", array(
        '!doclink' => $doclink,
      )),
```
This also includes a small wording change, BTW, "For more information see the...".

Reply:
Are you sure putting `t()` inside `t()` works properly...?

Reply:
I tested it locally, it works. At the time the !doclink placeholder is used, it's already populated with the full link, it's just markup.

Reply:
@indigoxela what you are suggesting is against what is advised re links within `t()`. See: https://www.drupal.org/node/322774

> Bad examples:

```php
// DO NOT DO THESE THINGS
$BAD_EXTERNAL_LINK = t('Look at Drupal documentation at !handbook.', array(
  '!handbook' => '<a href="http://drupal.org/handbooks">'. t('the Drupal Handbooks') .'</a>',
));
$ANOTHER_BAD_EXTERNAL_LINK = t('Look at Drupal documentation at <a href="http://drupal.org/handbooks">the Drupal Handbooks</a>.');
$BAD_INTERNAL_LINK = t('To get an overview of your administration options, go to !administer in the main menu.', array(
  '!administer' => l(t('the Administer screen'), 'admin'),
);
```

> Good examples:

```php
// Do this instead.
$external_link = t('Look at Drupal documentation at <a href="@drupal-handbook">the Drupal Handbooks</a>.', array(
  '@drupal-handbook' => 'http://drupal.org/handbooks',
));
$internal_link = t('To get an overview of your administration options, go to <a href="@administer-page">the Administer screen</a> in the main menu.', array(
  '@administer-page' => url('admin'),
));
```

Reply:
> what you are suggesting is against what is advised re links within t()

I disagree, because my suggestion obviously makes it easier for translators. If it's "against" something and may not be used... - it's only a suggestion. :wink:

BTW, my suggestion isn't actually covered by the "bad examples", as the variable containing the link is created beforehand and not within t().

Reply:
Thanks for the link @klonos. I recalled seeing that somewhere before, but couldn't find it yesterday...

@indigoxela I feel your suggestion is close enough to that 'bad' example that we shouldn't do it. Especially if only to avoid escaping a single apostrophe. Happy to hear from other core committees though.